### PR TITLE
update portal in qa-niaid

### PR DESCRIPTION
**Pull request by mfshao**
this is the NIAID branch rebased with `the real thing` 💣

### CODE REVIEW

File: qa-niaid.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -52,231 +52,29 @@
...
-        }
-      ]
-    },
-    "table": {
-      "enabled": false
+    "defaultOpenStudyName": "",
+    "dataType": "cmc",
+    "title": "Studies",
+    "titleField": "title",
+    "rowAccessor": "project_id",
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/portal/gitops.json:61]: can we use submitter_id instead

**Review by paulineribeyre — APPROVED** [review]

File: qa-niaid.planx-pla.net/portal/gitops.json (hunk 1 above)
Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/portal/gitops.json:61]: doesn't have to be right now

Comment (mfshao) [reply qa-niaid.planx-pla.net/portal/gitops.json:61]: yeah just need to get submitter_id into all 3 es indices

can do that in another pr?

Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/portal/gitops.json:61]: yeah!

**Pull request merged**